AcceptanceTests/AcceptanceTests.Common/Driver/Helpers/DriverExtensions.cs

```diff
@@ -39,8 +39,12 @@ public static IWebElement WaitUntilElementClickable(this IWebDriver driver, By e
         {
             try
             {
-                var wait = new WebDriverWait(driver, TimeSpan.FromSeconds(timeout));
-                return wait.Until(SeleniumExtras.WaitHelpers.ExpectedConditions.ElementToBeClickable(elementLocator));
+                WebDriverWait wait = new WebDriverWait(driver, timeout: TimeSpan.FromSeconds(30))
+                {
+                    PollingInterval = TimeSpan.FromSeconds(1),
```

Comment:
Same here for repeated code

Reply:
I'm going to convert this to a helper class as it's in two classes and may be used in more later.

Reply:
Isn't this already centralised/ put somewhere for common access?

Reply:
Ok.